### ISSUE TITLE
[Table Header] Ensure the `description` text provides sufficient contrast (CAL-AMP50) 

### ISSUE THREAD

**Issue by dqateam** [Esri/calcite-components#13541]
**monday.com sync:** #10821974752

### Violation: 
 Ensure text and images of text provide sufficient contrast 

### Recommended solution
_TBD_, requires design consultation regarding the `table-header`'s `description` color consideration.

<img width="702" height="291" alt="Image" src="https://github.com/user-attachments/assets/5bad8172-a22e-4b39-b98c-126a5367faac" />

### WCAG Reference: 
- [1.4.3 Contrast (Minimum)](https://www.w3.org/WAI/WCAG22/Understanding/contrast-minimum.html) 

### Severity: 
 Low 

### Media Type: 
 Color and Contrast 

--- 
### Areas for Remediation: 
- Title: CAL-AMP50 Ensure text and images of text provide sufficient contrast 
- Module: M7 Table and color picker 
- Finding ID: [CAL-AMP50](https://esri.hub.essentia11y.com/short-link/DVsnbzmOVyzP2JfF) 

**Issue**
There is text that does not meet the required minimum color contrast ratio with its background.

Foreground color: #6B6B6B
Background color: #C7EAFF
Contrast ratio: 4.22:1
Required contrast ratio for this content: 4.50:1

Examples of text with this contrast ratio include:
- Designated routes (in selected state)
- (all, summer, winter)
- (paved, gravel)
- (in miles).

**User Impact**
Users with low vision will have difficulty reading this content.

**Code Reference** 
 ```
<div class="description"><!--?lit$792339209$-->Designated routes</div>

```

**Steps to Reproduce**
1. Locate the text.
2. Inspect it with Chrome DevTools.
3. In the Styles tab, copy the text color and background color.
4. Use a calculator such as https://www.levelaccess.com/color-contrast-checker-new/ to determine the contrast ratio.
5. Notice that the contrast ratio is below the required value for this size of text.
 
**Recommendation**
Ensure text and images of text provide sufficient contrast. The following contrast ratios are required:

- Text smaller than 18 pt (24 px), or smaller than 14 pt (19 px) if bold, must have a color contrast ratio of 4.50:1 or more with adjacent colors.

- Text 18 pt (24 px) or larger, or 14 pt (19 px) or larger if bold, must have a color contrast ratio of 3.00:1 or more with adjacent colors.

Disabled controls that do not accept user interaction are exempt from this requirement.

To calculate color contrast ratios, use a tool such as the Level Access Accessible Color Picker Chrome extension: https://chrome.google.com/webstore/detail/accessible-color-picker/bgfhbflmeekopanooidljpnmnljdihld or the Color Contrast Checker: https://www.levelaccess.com/color-contrast-checker-new/

**Compliant Code Example**
```
<!-- text with a contrast ratio of 12.6:1 with the background color -->
<style>
 body {
 background-color: #FFFFFF;
 color: #333333;
 }
</style>

<h1>Welcome to our Website</h1>
<p>This is an example of non-compliant text with insufficient color contrast.</p>
```

 **Issue Image** 
 https://desktopqa/aic_lap/Calcite/Calcite2025/CAL-AMP50.png 
 

 --- 
### Additional Resources: 
- [LA Platform Report Link](https://esri.hub.essentia11y.com/short-link/DVsnbzmOVyzP2JfF) 
- [Esri Accessibility Resources](https://esriis.sharepoint.com/sites/Accessibility)
- Report Source: Product accessibility evaluation conducted on Calcite by Level Access and delivered August, 2025.

**Comment by DintaMel** [comment]
@geospatialem The `background-color` should be #f2f2f2 which would have enough contrast. Can you let me know if there's any action we need to take?

**Comment by geospatialem** [comment]
> [@geospatialem](https://github.com/geospatialem) The `background-color` should be #f2f2f2 which would have enough contrast. Can you let me know if there's any action we need to take?

Confirmed when checking `#6B6B6B` with `#F2F2F2` with https://webaim.org/resources/contrastchecker/?fcolor=6B6B6B&bcolor=F2F2F2, closing the above out as unplanned.
